packages/subgraph/src/utils.ts

```diff
@@ -242,22 +242,20 @@ export function getAmountStreamedSinceLastUpdatedAt(
 }
 
 /**
- * calculateMaybeCriticalAtTimestamp will return optimistic date based on updatedAtTimestamp, totalDeposit, balanceUntilUpdatedAt and totalNetFlowRate.
+ * calculateMaybeCriticalAtTimestamp will return optimistic date based on updatedAtTimestamp, balanceUntilUpdatedAt and totalNetFlowRate.
  * @param updatedAtTimestamp
  * @param balanceUntilUpdatedAt
- * @param totalDeposit
  * @param totalNetFlowRate
  */
 
 export function calculateMaybeCriticalAtTimestamp(
     updatedAtTimestamp: BigInt,
     balanceUntilUpdatedAt: BigInt,
-    totalDeposit: BigInt,
     totalNetFlowRate: BigInt
 ): BigInt {
     if (balanceUntilUpdatedAt.le(BIG_INT_ZERO)) return BIG_INT_ZERO;
     if (totalNetFlowRate.ge(BIG_INT_ZERO)) return BIG_INT_ZERO;
-    const criticalTimestamp = balanceUntilUpdatedAt.plus(totalDeposit).div(totalNetFlowRate.abs());
```

Comment:
I think `secondsUntilCritical` would be a better name.

Reply:
DONE